srcpkgs/Ice/template

```diff
@@ -1,86 +1,60 @@
 # Template file for 'Ice'
 pkgname=Ice
-version=3.5.1
-revision=22
-build_wrksrc=cpp
-makedepends="zlib-devel bzip2-devel mcpp-devel db-devel expat-devel libressl-devel"
+version=3.7.5
+revision=1
+wrksrc="ice-${version}"
+build_style=gnu-makefile
+make_build_args="LANGUAGES=cpp prefix=${DESTDIR}/usr OPTIMIZE=yes
+ USR_DIR_INSTALL=yes CONFIGS=shared CONFIGS+=cpp11-shared"
```

Comment:
`DESTDIR` isn't set during this step. I'm not sure how best to solve....

Reply:
Maybe try passing `install_libdir` here? https://github.com/zeroc-ice/ice/blob/3.7/config/Make.rules#L156

I hate Ice already >.<

Reply:
I might be wrong about DESTDIR, too.

Reply:
Tell me about it....
I'll give it a try.

Reply:
The magic word turned out to be `lib-suffix=` ..sigh.
adding DESTDIR seemed to be completely superfluous anyway not sure why I was doing it; removed.